pkg/compiler/vars.go

```diff
@@ -6,8 +6,11 @@ import (
 
 type varScope struct {
 	localsCnt int
-	arguments map[string]int
-	locals    []map[string]varInfo
+	arguments struct {
+		size   int
```

Comment:
`number`?
`argnum` and `argvals`? Or `argcnt` and `argvals`

Reply:
Fixed.